components/kyma-metrics-collector/pkg/testing/fixtures/static_providers.json

```diff
@@ -1,213 +1,329 @@
 {
   "azure": {
     "standard_a1_v2": {
-        "cpu_cores": 1,
-        "memory": 2,
-        "storage": 10,
-        "max_nics": 1
+      "cpu_cores": 1,
```

Comment:
Is there a document describing how to get the values in this file?

Reply:
The values are copied it from helm chart https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml.

The machine types are manually updated as needed. For example: https://github.com/orgs/kyma-project/projects/17/views/7?pane=issue&itemId=51280791